docs-ref-conceptual/install-azure-cli-windows.md

```diff
@@ -58,9 +63,12 @@ To install the Azure CLI using PowerShell, start PowerShell **as administrator**
    $ProgressPreference = 'SilentlyContinue'; Invoke-WebRequest -Uri https://aka.ms/installazurecliwindows -OutFile .\AzureCLI.msi; Start-Process msiexec.exe -Wait -ArgumentList '/I AzureCLI.msi /quiet'; Remove-Item .\AzureCLI.msi
    ```
 
-This will download and install the latest version of the Azure CLI for Windows. If you already have a version installed, the installer will update the existing version.
+This will download and install the latest 32-bit installer of the Azure CLI for Windows. If you prefer a 64-bit install, change URL to `https://aka.ms/installazurecliwindowsx64`. If the Azure CLI is already installed, the installer will overwrite the existing version.
 
-To install specific version, replace the `-Uri` argument with `https://azcliprod.blob.core.windows.net/msi/azure-cli-<version>.msi` with version segment changed. Available versions can be found at [Azure CLI release notes](./release-notes-azure-cli.md).
+To install a specific version, replace the `-Uri` argument with the URL described in [Specific version](#specific-version).  Here is an example of using the 32-bit installer of the Azure CLI version [2.51.0](/cli/azure/release-notes-azure-cli#august-01-2023) in PowerShell:
```

Comment:
`[Specific version](#specific-version)` is under `# [Microsoft Installer (MSI)](#tab/azure-cli)` tab, but not available under `# [Microsoft Installer (MSI) with Command](#tab/powershell)`, so the link will not be clickable.

Reply:
I change specific version to a include file, please see it works.

Reply:
In view, the link works just fine.  It is a bit odd taking the customer into another tab, but I'm going to merge this PR.